packages/clay-css/src/scss/variables/_multi-step-nav.scss

```diff
@@ -2,7 +2,7 @@ $multi-step-nav-margin-bottom: 15px !default;
 $multi-step-nav-padding-x: 15px !default;
 $multi-step-nav-padding-y: 0 !default;
 
-$multi-step-icon-bg: $gray-200 !default;
+$multi-step-icon-bg: $gray-300 !default;
```

Comment:
We don't need to change this here. The change in `src/scss/atlas/variables/_multi-step-nav.scss` is good enough. Lexicon has no definitions for the base theme.